bluesky/run_engine.py

```diff
@@ -1027,13 +1046,28 @@ def _logbook(self, msg):
 
     @asyncio.coroutine
     def _configure(self, msg):
+        if self._bundling:
+            raise IllegalMessageSequence(
+                "Cannot configure after 'create' but before 'save'"
+                "Aborting!")
```

Comment:
Yes. Maybe later we can relax this, but I think it makes sense for now. No currently-implemented scans would violate, and if one did, incorrect metadata would be recorded. I vote for generalizing this in a future, more comprehensive effort.